### DISCUSSION TITLE
Welcome to riscv-isa-manual Discussions!

### DISCUSSION THREAD

**Discussion by rpsene** [riscv/riscv-isa-manual#986] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by benhur1899** [comment]
Can _The RISC-V Instruction Set Manual Volume I: Unprivileged ISA_ be viewed as one single specification?

**Reply by wmat** [reply]
Is this what you're looking for? https://github.com/riscv/riscv-isa-manual/releases/download/riscv-isa-release-056b6ff-2023-10-02/unpriv-isa-asciidoc.pdf

**Reply by benhur1899** [reply]
Thanks a lot! But what I want to know is that if this volume book can be seen as one single RV specification? Or it just a compilation of RV specifications? What's the definition of a RV specification?

**Comment by gfavor** [comment]
Currently there is not one document that contains ALL ratified extensions.
There are the Unpriv and Priv documents, various not-yet-integrated ISA
extension specs, and various Non-ISA extension specs.  On the ISA side the
near-term goal is to integrate all ISA specs into two (Unpriv and Priv)
spec documents.

Regarding definition of a spec, each architecture extension (whether ISA or
Non-ISA) has a specification.  I would distinguish that from integrating
ISA specs together into ISA spec *documents*.  Although many people will
loosely refer to the Priv document, for example, as the "Priv spec" insofar
as it contains Privileged Architecture specs.

Greg




Greg

**Reply by benhur1899** [reply]
Thanks a lot! Got it!

**Comment by Zoro210** [comment]
In the translation process, after finding the leaf page, we will check for the SUM and MXR bit of m-status register. How can the supervisor access the mstatus?

**Comment by chili-chips-ba** [comment]
We are curious the hear your expert take on [this question](https://github.com/chili-chips-ba/wireguard-fpga/issues/5#issue-2651618418) about RISC-V support for misaligned accesses in the ISS context. 

Also is there a standardized designation to convey whether misaligned accesses are supported in hardware, or implemented via software traps? It is not `Zam`, is it?
